.gitignore

```diff
@@ -24,3 +24,4 @@ benchmark_outputs
 .mvn/timing.properties
 node_modules
 product-test-reports
+.vscode
```

Comment:
Is it a file or directory (or potentially both)? If only a directory, use `.vscode/`

Reply:
folder  .. I updated